src/deepsparse/loggers/async_logger.py

```diff
@@ -59,6 +59,12 @@ def log(self, identifier: str, value: Any, category: MetricCategories):
         )
         job_future.add_done_callback(_log_async_job_exception)
 
+    def __str__(self, level=0):
+        whitespace = "\t" * level
+        text = "\n" + whitespace + f"{self.__class__.__name__}:"
```

Comment:
quick nit as well, this whole line can be an f-string to avoid multiple concatenations (same to other instances of this)